GlobalFishingWatch/pelagos-client
# 385 categorized layers

Connects https://github.com/GlobalFishingWatch/pelagos-client/issues/385

To test:
- Go to advanced mode
- Go to edit mode
- Expand one of the animations
- Click on the [empty] value right of "Category:"
- Enter a new category name
- Press enter

The animation should now be moved to a new tab with the name you entered.
